Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -278,4 +278,11 @@ public struct _NameMap: ExpressibleByDictionaryLiteral {
     let n = Name(transientUtf8Buffer: raw)
     return jsonToNumberMap[n]
   }
+  
+  /// Returns all proto names
+  internal var names: [Name] {
```

Comment:
Is this still used?  Seems like you might be using a map and description in other places?

Reply:
Yes it's still being used in `init(allFieldsOf message: M)` for initializing a FieldMask containing all paths of a message type `M`